scripts/users_scrapper.ts

```diff
@@ -0,0 +1,42 @@
+import { DateTime } from 'luxon'
+import logger from '@adonisjs/core/services/logger'
+import SksUser from '#models/sks_user'
+
+const url = 'https://live.pwr.edu.pl/sks/sks-data.csv'
+
+export async function runScrapper() {
+  try {
+    const response = await fetch(url)
+    if (!response.ok) {
+      logger.error(`Failed to fetch data: ${response.status} ${response.statusText}`)
+      return
+    }
+
+    const usersData = await response.text()
+    const currentDateTime = DateTime.now().setZone('Europe/Warsaw')
+
+    const rows = usersData.trim().split('\n')
+    for (const row of rows) {
+      const [time, movingAverage21, activeUsers] = row.split(';')
+      const timestamp = currentDateTime.set({
+        hour: Number.parseInt(time.split(':')[0], 10),
+        minute: Number.parseInt(time.split(':')[1], 10),
+        second: 0,
+        millisecond: 0,
+      })
+
+      await SksUser.updateOrCreate(
```

Comment:
actually if you would use `updateOrCreateMany` and do an update in one bulk after the loop, you would stress the db much less :P

Reply:
but prolly not a real world visibile difference in this specific use case

Reply:
have no idea how i missed that, i was looking at this method in docs today
ig that's the reason we have code review :D

Reply:
btw i see, that you make a quick progress in Adonis, maybe u want to join my backend team ;P